src/main/kotlin/app/revanced/patches/youtube/layout/spoofappversion/bytecode/patch/SpoofAppVersionPatch.kt

```diff
@@ -0,0 +1,64 @@
+package app.revanced.patches.youtube.layout.spoofappversion.bytecode.patch
+
+import app.revanced.extensions.toErrorResult
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.addInstructions
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.DependsOn
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patches.shared.settings.preference.impl.StringResource
+import app.revanced.patches.shared.settings.preference.impl.SwitchPreference
+import app.revanced.patches.youtube.layout.spoofappversion.annotations.SpoofAppVersionCompatibility
+import app.revanced.patches.youtube.layout.spoofappversion.bytecode.fingerprints.SpoofAppVersionFingerprint
+import app.revanced.patches.youtube.misc.integrations.patch.IntegrationsPatch
+import app.revanced.patches.youtube.misc.settings.bytecode.patch.SettingsPatch
+import org.jf.dexlib2.iface.instruction.OneRegisterInstruction
+
+@Patch
+@DependsOn([IntegrationsPatch::class, SettingsPatch::class])
+@Name("spoof-app-version")
+@Description("Makes YouTube think the app is an older version. This gives the old UI layout, but may cause unknown side effects.")
```

Comment:
```suggestion
@Description("Tricks YouTube into thinking, you are running an older version of the app. An (un)intended side effect is the former design of YouTube.")
```

Reply:
It's not unintended for the patch itself, because the intention is to give the old UI.

Reply:
This is why it's "(un)intended", not "unintended". Maybe:

```suggestion
@Description("Tricks YouTube into thinking, you are running an older version of the app. A side effect is the former design of YouTube.")
```

Reply:
How about:

"Tricks YouTube into thinking, you are running an older version of the app. This restores the old UI layout, but may cause unknown side effects."

The wording 'former design of youtube' may not convey it's the entire UI that changes.

I think the description should have a disclaimer about unknown effects, because it's not conclusively known that it will always work 100% correctly.  Could new features behave differently?  Could the app crash because it's served up old data it doesn't understand?  All are possible.

Reply:
In that case, it can be said "One of the side-effects..." which allows getting rid of the last sentence module "but may cause unknown side effects"

```suggestion
@Description("Tricks YouTube into thinking, you are running an older version of the app. One of the side effects also includes restoring the old UI.")
```